.github/workflows/build-base-image.yaml

```
@@ -0,0 +1,41 @@
+      - "Dockerfile"
+      - ".github/workflows/build-base-image.yaml"
+
+jobs:
+  build:
+    name: Build & push to ECR
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

actionlint (1.7.12)

[error] the runner of "aws-actions/configure-aws-credentials@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)